src/styles.scss

```diff
@@ -795,6 +795,17 @@ table.match-table tr {
 
 .rankings-table td, .rankings-table th {
   padding: .75rem;
+
+  .mdc-icon-button {
+    height: 24px;
+    width: 36px;
```

Comment:
```suggestion
    --mdc-ripple-fg-scale: 1.2 !important;
    height: 24px;
    width: 34px;
```

Reply:
Nice, the smaller size is better, thanks.
I also adjusted the margin a tiny amount so that header text adjacent to a button is lined-up vertically with the header text of other non-button columns.